sympy/concrete/expr_with_limits.py

```diff
@@ -25,12 +26,12 @@ def _process_limits(*symbols):
     limits = []
     orientation = 1
     for V in symbols:
-        if isinstance(V, Symbol):
+        if isinstance(V, (Symbol, Idx)):
```

Comment:
That's a good point. Yes, that can be done. But something like this should work?

```
>>> i = Idx('i', range=(1, 4))
>>> A = IndexedBase('A')
>>> Sum(A[i], (i, 1, 2))
A[1] + A[2]
``